### ISSUE TITLE
GraphQL v4: Child pages not able to be viewed in DRAFT 

### ISSUE THREAD

**Issue by blueo** [silverstripe/silverstripe-graphql#349]
We're querying child pages we're not able to view pages that are in DRAFT via graphql eg this query:

```
query ReadTilePage($id: ID!, $stage: VersionedQueryMode!) {
  readOneSiteTree(filter: { id: { eq: $id } }, versioning: { mode: $stage }) {
    id
    metaDescription
    title
    content
    children {
      id
      title
      content
      link
      _extend {
        cardPage {
          icon
        }
      }
    }
    __typename
  }
}
```

with the stage set to DRAFT we're only getting the published pages. 

Its a fairly simple schema to reproduce it:

```
Page:
  fields:
    "*": true
  operations:
    read: true
 ```

**Comment by chillu** [comment]
@unclecheese I think you worked through this with Bernie separately?

**Comment by blueo** [comment]
@chillu we had a chat but it is still unresolved FYI

**Comment by unclecheese** [comment]
On my list to do next

**Comment by unclecheese** [comment]
Resolved in one of the commits to a forthcoming PR: https://github.com/silverstripe/silverstripe-versioned/pull/319/commits/11615a3444174b4aaadf1f8f1ee28144c6ee7efa

**Comment by unclecheese** [comment]
Merged, in 4.0.0-alpha 1 https://github.com/silverstripe/silverstripe-graphql/releases/tag/4.0.0-alpha1
